must-gather/gather-power-monitoring-operator-info

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# copyright 2023.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+#
+
+set -eu -o pipefail
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "${SCRIPT_DIR}/common"
+
+BASE_COLLECTION_PATH=$1
+
+KEPLER_OPERATOR_NS="openshift-operators"
+
+get_catalogsource() {
+	log "getting catalogsource info for kepler-operator"
+	run oc -n "$KEPLER_OPERATOR_NS" get catalogsource kepler-operator-catalog -oyaml "$POWER_MONITORING_OPERATOR_INFO_DIR/kepler-operator-catalogsource.yaml"
```

Comment:
Shouldn't we remove this as the operator would be from Red Hat certified Operators catalog source right?

Reply:
kept it for community operator

Reply:
but `gather-power-monitoring-operator-info` will never be called up for community operator